### PR TITLE
Adding info about overwriteWithBlank for batch endpoint

### PR DESCRIPTION
**Pull request by escopecz**
Related to https://github.com/mautic/mautic/pull/12255

<a href="https://gitpod.io/#https://github.com/mautic/developer-documentation-new/pull/122"></a>

### CODE REVIEW

File: docs/rest_api/contacts.rst [hunk 1]
```diff
@@ -604,15 +604,18 @@ Edit Batch Contact
...
            'email'     => 'jim@example.com',
            'ipAddress' => $_SERVER['REMOTE_ADDR']
        ],
        [
-           'id'        => 1,
-           'firstname' => 'Ashish',
-           'lastname'  => 'Wallach',
-           'email'     => 'ashish@example.com',
-           'ipAddress' => $_SERVER['REMOTE_ADDR']
+           'overwriteWithBlank' => true, // This flag will allow to overwrite any field with a blank value
```
Comment (RCheesley) [inline docs/rest_api/contacts.rst:612]:
```suggestion
           'overwriteWithBlank' => true, // This flag will allow you to overwrite any field with a blank value
```

**Review by RCheesley — CHANGES_REQUESTED** [review]
Just a minor grammar suggestion!

File: docs/rest_api/contacts.rst (hunk 1 above)
Comment (escopecz) [reply docs/rest_api/contacts.rst]: Thanks! Updated.

**Review by RCheesley — APPROVED** [review]
GTG!

**Pull request merged**